src/averageAges.js

```diff
@@ -15,6 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  let man = [];
+
+  if (!century) {
+    man = people.filter(person => person.sex === 'm');
+  } else {
+    man = people.filter(person =>
+      person.sex === 'm' && Math.ceil(person.died / 100) === century);
+  }
+
+  const sum = man.reduce((prev, person) =>
+    prev + (person.died - person.born), 0);
```

Comment:
How about creating additional function for calculating age sum, that would work for both `men` and `women`:
`calculateAgeSum(filteredGroup)`, (you can choose your own names)